src/lib/Browsers.js

```diff
@@ -4,6 +4,10 @@ const semverCompare = require('semver-compare')
 const BrowserList = function () {
   this.browsers = Object.keys(compatData.browsers).reduce((browsers, browser) => {
     browsers[browser] = Object.keys(compatData.browsers[browser].releases)
+      .filter(function (release) {
+        const status = compatData.browsers[browser].releases[release].status
```

Comment:
I wonder if it would make sense to add some safeguards to this check, in case one of the nodes in the chain (e.g. `compatData.browsers[browser].releases[release]`) does not exist. May be redundant, but I did come across some issues with inconsistent formats in the data that would cause issues when reading.